### DISCUSSION TITLE
Any recommandation to improve the efficiency of DP model in MD simulations?

### DISCUSSION THREAD

**Discussion by Johnsyisme** [deepmodeling/deepmd-kit#3719] [Q&A]
I'm doing a 3-elements systems with a 3 layers(240-240-240) se_se_a model using type embeding network (2-4-8).
I just want to improve the speed of MD simulations, and I'm not sure which hyperparameter has the biggest impact on MD efficiency.

Here is what came to my mind:
1.  I have tried the attention-based descriptor, it is much more slower (I'm not sure if this is normal [running on A100/H100]).
2. Decrease the cutoff ()
3. Decrease the nodes of each layer
4. Compress the model (keyError 'layer_0' when i compress the model using type_embedding net)
Since I assume that the most time consumming part of DP model may be the coordinate->descriptor part instead of the neural net part, I'm not quite sure which of the above approaches should be the largest.

If I could trade partial accuracy for computational efficiency, which one might be the best:
1. compressed atten_v2
2. type_embedding but with smaller cutoff and less nodes
3. Stop using type_embedding and compress the model.
4. using other model like se_e2_r

Thank you very much!!!

**Comment by wanghan-iapcm** [comment]
please try the compressible setting of the dpa-1 model (without attention layers). 
https://github.com/deepmodeling/deepmd-kit/blob/r2/examples/water/se_atten_compressible/input.json

**Reply by Johnsyisme** [reply]
Appreciate !
